### ISSUE TITLE
documentation error for application_gateway

### ISSUE THREAD

**Issue by jaekim09** [terraform-providers/terraform-provider-azurerm#3113]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Terraform (and AzureRM Provider) Version

```code
$ terraform -v
Terraform v0.11.11
+ provider.azurerm v1.23.0
+ provider.external v1.1.0
```

### Affected Resource(s)

* azurerm_application_gateway's documentation

### Terraform Configuration Files
```hcl
http2_enabled = [true | false]
```
### Expected Behavior

It should configure http2 on Application Gateway

### Actual Behavior

```code
$ terraform plan

Error: module.WAF.azurerm_application_gateway.waf: : invalid or unknown key: http2_enabled
```

### Steps to Reproduce

1. in configuration, use key http2_enabled. 
2, terraform plan/apply

### What it should be
the key suppose to be enable_http2. 
the documentation should be updated to enable_http2.

related comment from issue 1576
https://github.com/terraform-providers/terraform-provider-azurerm/issues/1576#issuecomment-466819669

* #0000

**Comment by Lucretius** [comment]
This has been resolved in https://github.com/terraform-providers/terraform-provider-azurerm/pull/3095 and should be out in the next release.

**Comment by tombuildsstuff** [comment]
hey @jaekim09 

Thanks for opening this issue :)

As @Lucretius has mentioned, this issue has previously been reported in #3030 (and has been fixed in #3095) which will be released as a part of the next release. Since the fix for this has been merged I'm going to close this issue for the moment, but thanks for taking the time to report this - and as mentioned above this should be deployed in the near future 👍 

Thanks!
